pkg/services/postgresql/catalog.go

```diff
@@ -103,6 +103,17 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 					Bullets:     []string{"100 DTUs"},
 				},
 			}),
+			service.NewPlan(&service.PlanProperties{
+				ID:          "ba653f35-2152-4d76-8641-b21d4636b2e1",
+				Name:        "standard100",
+				Description: "Standard Tier, 100 DTUs",
+				Free:        false,
+				Extended: map[string]interface{}{
+					"skuName":        "PGSQLS100",
+					"skuTier":        "Standard",
+					"skuCapacityDTU": 100,
```

Comment:
note I am still confirming these are the right `sku{Name,Tier,CapacityDTU}`

Reply:
You will need to update the ARM template as well. We constrain valid values in the template right now.

Reply:
got it, thanks @jeremyrickard - commit incoming

Reply:
done in https://github.com/Azure/open-service-broker-azure/pull/390/commits/284e4b7c6e850c9918cad7ce3bcd30e6418b8109